src/conventional/sonde_tac2ioda.py

```diff
@@ -966,8 +970,12 @@ def change_vars(profile):
         if 'height' in levels[pressure] and levels[pressure]['height'] is not None:
             height = levels[pressure]['height']
             dz = height - heightKm1
-            # Typical radiosonde ascent rate is 5 m/s
-            this_datetime = previous_time + timedelta(seconds=dz*0.2)
+            # Legacy soundings produce null values at mandatory level below ground.
+            if (dz < 1.0):
```

Comment:
Can I check why the threshold of `1.0` is used here please?

Reply:
Simply used a starting value of 1 to ensure the balloon is "lifting" away from the ground.  It's arbitrary.  In truth, we never seem to know exactly what elevation a real launch happens given the accuracy in most station tables.